core/src/test/scala/org/apache/spark/util/ClosureCleanerSuite.scala

```diff
@@ -50,7 +50,7 @@ class ClosureCleanerSuite extends FunSuite {
     val obj = new TestClassWithNesting(1)
     assert(obj.run() === 96) // 4 * (1+2+3+4) + 4 * (1+2+3+4) + 16 * 1
   }
-  
+
```

Comment:
The suite should be expanded with the fix, right? Could you also test the common `for` loop case that others have seen in the past?